source/MaterialXRender/Handlers/ImageHandler.cpp

```diff
@@ -1,38 +1,78 @@
+#include <MaterialXCore/Types.h>
+#include <MaterialXGenShader/Util.h>
 #include <MaterialXRender/Handlers/ImageHandler.h>
 #include <cmath>
 
 namespace MaterialX
 {
+bool ImageHandler::saveImage(const std::string& fileName,
+                            const ImageDesc &imageDesc)
+{
+    for (auto loader : _imageLoaders)
+    {
+        if (loader->supportsExtension(getFileExtension(fileName)))
+        {
+            return loader->saveImage(fileName, imageDesc);
+        }
+    }
+    return false;
+}
 
-bool ImageHandler::createDefaultImage(unsigned int& width,
```

Comment:
This method was just create some arbitrary texture. New more useful method is solid color image creator.